### ISSUE TITLE
[Bug] cuda断言

### ISSUE THREAD

**Issue by peach-gif** [happytianhao/TaDe#1]
### Prerequisite

- [x] I have searched [Issues](https://github.com/open-mmlab/mmdetection3d/issues) and [Discussions](https://github.com/open-mmlab/mmdetection3d/discussions) but cannot get the expected help.
- [x] I have read the [FAQ documentation](https://mmdetection3d.readthedocs.io/en/latest/notes/faq.html) but cannot get the expected help.
- [x] The bug has not been fixed in the [latest version (dev-1.x)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.x) or [latest version (dev-1.0)](https://github.com/open-mmlab/mmdetection3d/tree/dev-1.0).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

main branch https://github.com/open-mmlab/mmdetection3d

### Environment

PyTorch: 2.1.0+cu121
Torchvision: 0.16.0+cu121
CUDA available: True
CUDA version: 12.1
Python 3.8.5

### Reproduces the problem - code sample

源代码没变CUDA_VISIBLE_DEVICES=0,1 nohup bash tools/dist_train.sh     projects/TaDe/configs/end2end.py     2     --work-dir work_dirs/tade_experiment     > train_log_final.out 2>&1 &

### Reproduces the problem - command or script

CUDA_VISIBLE_DEVICES=0,1 nohup bash tools/dist_train.sh     projects/TaDe/configs/end2end.py     2     --work-dir work_dirs/tade_experiment     > train_log_final.out 2>&1 &

### Reproduces the problem - error message

01/20 21:39:31 - mmengine - [4m[37mINFO[0m - Epoch(train)  [1][ 1700/28008]  lr: 5.0000e-04  eta: 7 days, 16:55:09  time: 0.5911  data_time: 0.0066  memory: 8057  grad_norm: 0.2195  loss: 0.4425
../aten/src/ATen/native/cuda/Loss.cu:94: operator(): block: [32,0,0], thread: [64,0,0] Assertion `input_val >= zero && input_val <= one` failed.
../aten/src/ATen/native/cuda/Loss.cu:94: operator(): block: [32,0,0], thread: [65,0,0] Assertion `input_val >= zero && input_val <= one` failed.
../aten/src/ATen/native/cuda/Loss.cu:94: operator(): block: [32,0,0], thread: [66,0,0] Assertion `input_val >= zero && input_val <= one` failed.
../a出现错误，如何修改呢

**Comment by peach-gif** [comment]
运行Tade这个项目出现问题
